aws_s3_bucket_policy.tf

```diff
@@ -5,109 +5,37 @@
 resource "aws_s3_bucket_policy" "allowActionsFromAllPrincipals" {
   bucket = "test"
 
-  policy = <<POLICY
-{
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket should not allow all actions from all principals**
 
**Vulnerability**
Misconfigured S3 buckets can leak private information to the entire internet or allow unauthorized data tampering / deletion
 
**Remediation**
In the S3 console, select the Permissions tab, and then Bucket Policy. Remove policies for s3:* actions for principals '*'. If necessary, modify the policy instead, to limit the access to specific principals. Refer to: https://docs.aws.amazon.com/AmazonS3/latest/dev/using-iam-policies.html

source: accurics-